pysam/__init__.py

```diff
@@ -43,9 +43,26 @@
 def get_include():
     '''return a list of include directories.'''
     dirname = os.path.abspath(os.path.join(os.path.dirname(__file__)))
-    return [dirname,
-            os.path.join(dirname, 'include', 'htslib'),
-            os.path.join(dirname, 'include', 'samtools')]
+
+    #
+    # Header files may be stored in different relative locations
+    # depending on installation mode (e.g., `python setup.py install`,
+    # `python setup.py develop`. The first entry in each list is
+    # where develop-mode headers can be found.
+    #
+    htslib_possibilities = [os.path.join(dirname, '..', 'htslib'),
+                            os.path.join(dirname, 'include', 'htslib')]
```

Comment:
Turns out, in my first attempt to support develop-mode I did exactly as you suggested. Somewhat confusingly, though, _htslib_ header files are stored in a subdirectory called `htslib`, twice... For example, when you run `python setup.py develop`, htslib headers can be found in `<$PKG_ROOT/htslib/htslib/hts.h>`, while when you run `python setup.py install` they live in, e.g., `$YOUR_VE/lib/python2.7/site-packages/pysam/include/htslib/htslib/hts.h`.